frontend/Makefile

```diff
@@ -31,6 +31,28 @@ endif
 push: image
 	docker push ${ARO_HCP_FRONTEND_IMAGE}
 
+
+kustomize-update:
+	@test "${RESOURCEGROUP}" != "" || (echo "RESOURCEGROUP must be defined" && exit 1)
```

Comment:
For convenience, can we have a default resource group that matches dev-infrastructure?